src/models/m_rsc.erl

```diff
@@ -120,7 +120,7 @@ m_value(#m{value=Id}, Context) ->
     get_visible(Id, Context).
 
 %% @doc Return the id of the resource with the name
-% @spec name_to_id(NameString, Context) -> {ok, int()} | {error, Reason}
+-spec name_to_id(string(), #context{}) -> {ok, int()} | {error, Reason}.
```

Comment:
Maybe we should have a more generic type for resource names, as we accept: `atom() | binary() | string()`